oak_attestation_verification/src/verifier.rs

```diff
@@ -90,7 +126,7 @@ fn verify_oak_containers(
     _endorsements: &OakContainersEndorsements,
     _reference_values: &OakContainersReferenceValues,
 ) -> anyhow::Result<()> {
-    anyhow::bail!("Needs implementation")
+    Ok(()) // Work in progress
```

Comment:
Could you instead add a `// TODO(#4074): ...`

Reply:
Done.